app/locale/en.coffee

```diff
@@ -138,6 +138,7 @@
     next: "Next"  # Go from choose hero to choose inventory before playing a level
     change_hero: "Change Hero"  # Go back from choose inventory to choose hero
     change_hero_or_language: "Change Hero or Language"
+    change_hero: "Change Hero"
```

Comment:
Is there a reason for a second `change_hero`?

Reply:
Yes, students should not be able to change language, they get the option to change just the hero selected!

Reply:
We shouldn't need to translate this twice.  And, if we really wanted to translate the same string twice, we'd still need to use different keys.  Both are `change_hero`, so I doubt they both get propagated anyway.

Reply:
This is a merge issue. Sorry! When I saw Andrew's comment I thought he is asking about change_hero_or_language and change_hero